[Security Solution][Explore][Tests] Migrate from enzyme to rtl

kapral18 opened the pull request:
addresses #217294

This PR migrates all existing enzyme usage to react-testing-library. This PR is not attempting to refactor or move from unit testing to integration style testing with less mocks. In fact I add even more mocks in some places where after moving out of enzyme things start to blow up because enzyme structure was hiding the lack of mocks.

Here is essence of the change:

- all shallow toMatchSnapshot tests have been migrated to render() toMatchSnapshot() tests which has caused an expected spike in amount of changes. For now I am leaving it as is. Later we can decide on what to do (either selective mock underlying implementations to shrink snapshot sizes or cut them altogether in favor of specific element existence assertions

- some dead tests were cut that were passing with enzyme. Enzyme is testing the component tree not the dom tree giving a false sense of stuff being rendered when it's not, that's why we have to cut it out.

- in some places I use `expect(container.children[0]).toMatchSnapshot()` as a viable alternative for not introducing an unnecessary `data-test-subj` locator on the root element. The result is the same.

- wherever possible I replaced selectors usage in tests to `screen.getByTestId` and where not to `container.querySelector` or `document.querySelector`(for global selection)

- other matchers were transitioned as is

### A reminder on how to quickly fast track through the migration:

1. use `expect(container.children[0]).toMatchSnapshot()` when you need to find alternative for root element snapshotting without introducing a `data-test-subj`. Especially for when it's not as self-evident. Gets job done in 99% of cases I faced.

2. when you need to transition with deeply mocked components and testing component prop shapes  into RTL land you can transition it into RTL like this

For example

```jest.mock('@elastic/charts', () => {
  const actual = jest.requireActual('@elastic/charts');

  return {
    ...actual,
    AreaSeries: jest.fn(() => <div data-test-subj="area-series-mock" />),
    Axis: jest.fn(() => <div data-test-subj="axis-mock" />),
  };
});

const MockedAreaSeries = AreaSeries as jest.MockedFunction<typeof AreaSeries>;
const MockedAxis = Axis as jest.MockedFunction<typeof Axis>;

it('should render AreaSeries with given yScaleType', () => {
   expect(MockedAreaSeries.mock.calls[0][0].yScaleType).toEqual(configs.series.yScaleType);
});

it('should render xAxis with given tick formatter', () => {
  expect(MockedAxis.mock.calls[0][0].tickFormat).toEqual(mockTimeFormatter);
});

// or use 

expect(MockedAxis).toHaveBeenCalledWith(expect.objectContainer(...)) alternative

```

3. For matchers that use css selectors you can use `container.querySelector` or `container.querySelectorAll` or `document.querySelector` for global search (for modals and popovers)

4. If you have something like `wrapper.find('tbody tr td a').at(3).find('div span').at(2).text()` => `container.querySelector('tbody tr td a:nth-child(4) div span:nth-child(3)')?.textContent)`

5. Use https://kertal.github.io/codeowner-filter/ to quickly filter through codeowned files if you use vscode

6. If you happen to use `container.querySelectorAll` or `document.querySelectorAll()` and want to target the last element you can actually turn the NodeList into array and use array methods like expect(Array.from(container.querySelectorAll(...)).at(-1)).toHaveContext('foobar')

7. When you see selector like `wrapper.find('[data-test-subj="a"] [data-test-subj="b"]')` you can still use rtl DSL by using within => `within(screen.getByTestId('a')).getByTestId('b')`

<!--ONMERGE {"backportTargets":["8.19"]} ONMERGE-->

kapral18 committed:
chore: migrate tests to React Testing Library part 1

- accessibility tooltip with keyboard shortcut tests
- charts areachart tests
- charts draggable legend item tests

kapral18 committed:
chore: migrate tests to React Testing Library part 2

- Add data-test-subj attributes to AreaChartBaseComponent and
BarChartBaseComponent because RenderResult.container is not part of the
render tree
- Refactor areachart and barchart tests to use @testing-library/react
instead of enzyme
- Add beforeEach hooks to clear mocks before each test run and render
  components because implicit cleanup() is nuking render tree between
  tests if we use beforeAll()
- Update ChartPlaceHolder tests to use @testing-library/react and check
for styles using @emotion/jest matchers

kapral18 committed:
chore: migrate tests to React Testing Library part 3

- charts common tests
- header page index
- header page title
- header section index
- subtitle index

kapral18 committed:
chore: migrate tests to React Testing Library part 4

- inspect index
- links index

kapral18 committed:
chore: migrate tests to React Testing Library part 5

- matrix-histogram index
- navigation tab_navigation
- tables helpers
- top_n index
- top_n top_n

kapral18 committed:
chore: migrate tests to React Testing Library part 6

- tab_navigation
- paginated_table
- hosts_table
- pages/hosts
- flow_target_select_connected
- pages/network
- pages/users_tabs

elasticmachine commented:
Pinging @elastic/security-threat-hunting-explore (Team:Threat Hunting:Explore)

elasticmachine commented:
Pinging @elastic/security-threat-hunting (Team:Threat Hunting)

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292839)
* Commit: b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/292839#019634cd-21ee-46d0-91a8-03df51209028)

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":292839}
buildkite-pr-comment-->

kapral18 committed:
chore: migrate tests to React Testing Library part 7

- uncommon_process_table
- arrow
- details
- embeddables/embeddable
- flow_controls/flow_target_select
- ip
- kpi_network
- network_dns_table
- network_dns_table/is_ptr_included
- network_http_table
- network_top_countries_table
- network_top_n_flow_table

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292959)
* Commit: b851188b5aaa36065bfdc5beb57e1526cbdeaeae

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/292959#01963645-aa80-4917-a992-bc904b6b58b6)

### History
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":292959}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292975)
* Commit: 93c362649e94387fa2a7513f1bec203ce6956ea1

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/292975#01963661-e304-4030-93e2-7a059691a0c2)

### History
* :broken_heart: [Build #292959](https://buildkite.com/elastic/kibana-pull-request/builds/292959) failed b851188b5aaa36065bfdc5beb57e1526cbdeaeae
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":292975}
buildkite-pr-comment-->

kapral18 committed:
chore: migrate tests to React Testing Library part 8

- embeddables
- embeddables/index_patterns_missing_prompt
- embeddables/map_tool_tip/line_tool_tip_content
- embeddables/map_tool_tip/point_tool_tip_content
- embeddables/tooltip_footer
- flow_controls/flow_direction_select
- port
- tls_table
- user_table
- pages/overview

kapral18 committed:
chore: migrate tests to React Testing Library part 9

- empty_page
- event_counts
- host_overview/endpoint_overview
- host_overview
- overview_cti_links
- overview_host
- overview_host_stats
- overview_network
- overview_network_stats
- user_overview

kapral18 committed:
chore: fix ci regressions

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293147)
* Commit: e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/293147#0196393f-8155-476d-8ca3-58706d5ffefd)

### History
* :broken_heart: [Build #292975](https://buildkite.com/elastic/kibana-pull-request/builds/292975) failed 93c362649e94387fa2a7513f1bec203ce6956ea1
* :broken_heart: [Build #292959](https://buildkite.com/elastic/kibana-pull-request/builds/292959) failed b851188b5aaa36065bfdc5beb57e1526cbdeaeae
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":293147}
buildkite-pr-comment-->

agusruidiazgd reviewed (approved):
LGTM, thanks Karen for this changes 🚀

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293177)
* Commit: b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35

### Failed CI Steps
* [FTR Configs #99](https://buildkite.com/elastic/kibana-pull-request/builds/293177#01963986-60e7-4cee-95fc-a2860a2abd43)
* [Serverless Rule Management - Security Solution Cypress Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/293177#01963964-c2c8-4c3c-9d57-08e2c2bd6675)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293177#01963986-60e7-4cee-95fc-a2860a2abd43) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293177/jobs/01963986-60e7-4cee-95fc-a2860a2abd43/artifacts/0196399d-2eb3-4e17-af5a-aa4a3e14b662) FTR Configs #<span></span>99 / Cloud Security Posture Findings Page - Alerts Create detection rule Creates a detection rule from the Take Action button and navigates to rule page
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293177#01963964-c2c8-4c3c-9d57-08e2c2bd6675) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293177/jobs/01963964-c2c8-4c3c-9d57-08e2c2bd6675/artifacts/019639a5-b444-4b9b-a0d2-d087efd815ac) Serverless Rule Management - Security Solution Cypress Tests #<span></span>1 / Related integrations integrations installed (AWS CloudFront (enabled), AWS CloudTrail (disabled), System (enabled)) rule details "before each" hook for "should display the integrations in the definition section" "before each" hook for "should display the integrations in the definition section"

### History
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a
* :broken_heart: [Build #292975](https://buildkite.com/elastic/kibana-pull-request/builds/292975) failed 93c362649e94387fa2a7513f1bec203ce6956ea1
* :broken_heart: [Build #292959](https://buildkite.com/elastic/kibana-pull-request/builds/292959) failed b851188b5aaa36065bfdc5beb57e1526cbdeaeae
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":293177}
buildkite-pr-comment-->

kapral18 committed:
chore: fix ci regressions part 2

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293499)
* Commit: 751a14756e426f39e1cc301279aba4c4ea2696e5

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/293499#01963bd1-0697-40a0-89a6-ddd6d0190f4d)

### History
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a
* :broken_heart: [Build #292975](https://buildkite.com/elastic/kibana-pull-request/builds/292975) failed 93c362649e94387fa2a7513f1bec203ce6956ea1
* :broken_heart: [Build #292959](https://buildkite.com/elastic/kibana-pull-request/builds/292959) failed b851188b5aaa36065bfdc5beb57e1526cbdeaeae
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292839","number":292839,"commit":"b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0"}],"number":293499}
buildkite-pr-comment-->

janmonschke commented on x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_kpis/alerts_histogram_panel/index.test.tsx:
```diff
@@ -530,7 +530,9 @@ describe('AlertsHistogramPanel', () => {
       wrapper.update();
 
-      expect(wrapper.find(`[data-test-subj="header-section-subtitle"]`).text()).toContain('999');
+      expect(wrapper.find(`[data-test-subj="header-section-subtitle"]`).first().text()).toContain(
```
@kapral18 It seems like this file is still using enzyme. Is that because it's out of scope for the explore team?

kapral18 committed:
chore: fix ci regressions part 3

kapral18 committed:
chore: restrict usage on enzyme imports in explore folders

janmonschke reviewed (approved):
THI changes lgtm

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293709)
* Commit: 82472eca44c309d1e263830e9914c16da0c40a8f
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-218134)

### Failed CI Steps
* [Jest Tests #14](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-949d-4bee-961a-a5970df14245)
* [FTR Configs #29](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-94b2-40d0-ac41-204bbb22636b)
* [FTR Configs #29](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963ed3-acd0-400d-9c45-1d1ce8289ffc)
* [FTR Configs #107](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-94e7-49f7-9900-e0debf9ed245)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-94b2-40d0-ac41-204bbb22636b) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293709/jobs/01963eac-94b2-40d0-ac41-204bbb22636b/artifacts/01963ed3-8067-4774-9624-f6a62fd4ded6) FTR Configs #<span></span>29 / @<span></span>ess @<span></span>serverless SecuritySolution Explore Overview Overview Host With auditbeat Make sure that we get OverviewHost data
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963ed3-acd0-400d-9c45-1d1ce8289ffc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293709/jobs/01963ed3-acd0-400d-9c45-1d1ce8289ffc/artifacts/01963edc-88f3-497d-a829-2544aa688585) FTR Configs #<span></span>29 / @<span></span>ess @<span></span>serverless SecuritySolution Explore Overview Overview Host With auditbeat Make sure that we get OverviewHost data
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-94e7-49f7-9900-e0debf9ed245) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293709/jobs/01963eac-94e7-49f7-9900-e0debf9ed245/artifacts/01963ed5-2a4a-4a93-a721-d53985ce799f) FTR Configs #<span></span>107 / console app misc console behavior keyboard shortcuts open documentation should open documentation when Ctrl+/ is pressed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-949d-4bee-961a-a5970df14245) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293709/jobs/01963eac-949d-4bee-961a-a5970df14245/artifacts/01963eda-4654-4ab3-ba50-8c93f0e96051) Jest Tests #<span></span>14 / HeaderSection it renders
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/293709#01963eac-949d-4bee-961a-a5970df14245) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/293709/jobs/01963eac-949d-4bee-961a-a5970df14245/artifacts/01963eda-4654-41e5-930f-a9efb3807f33) Jest Tests #<span></span>14 / Subtitle it renders

### History
* :broken_heart: [Build #293499](https://buildkite.com/elastic/kibana-pull-request/builds/293499) failed 751a14756e426f39e1cc301279aba4c4ea2696e5
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a
* :broken_heart: [Build #292975](https://buildkite.com/elastic/kibana-pull-request/builds/292975) failed 93c362649e94387fa2a7513f1bec203ce6956ea1
* :broken_heart: [Build #292959](https://buildkite.com/elastic/kibana-pull-request/builds/292959) failed b851188b5aaa36065bfdc5beb57e1526cbdeaeae
* :broken_heart: [Build #292839](https://buildkite.com/elastic/kibana-pull-request/builds/292839) failed b1dfb4a0a53674f6c0f9ab6ee141d46a4543fbd0

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293709","number":293709,"commit":"82472eca44c309d1e263830e9914c16da0c40a8f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292959","number":292959,"commit":"b851188b5aaa36065bfdc5beb57e1526cbdeaeae"}],"number":293709}
buildkite-pr-comment-->

kapral18 committed:
chore: restrict usage on enzyme imports in explore folders (attempt 2)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294108)
* Commit: 290ca495abe74643730be395018259ac1f4f5ee3

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/294108#01964373-0226-4726-b4d6-4ee3a360555f)

### History
* :broken_heart: [Build #293709](https://buildkite.com/elastic/kibana-pull-request/builds/293709) failed 82472eca44c309d1e263830e9914c16da0c40a8f
* :broken_heart: [Build #293499](https://buildkite.com/elastic/kibana-pull-request/builds/293499) failed 751a14756e426f39e1cc301279aba4c4ea2696e5
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a
* :broken_heart: [Build #292975](https://buildkite.com/elastic/kibana-pull-request/builds/292975) failed 93c362649e94387fa2a7513f1bec203ce6956ea1

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294108","number":294108,"commit":"290ca495abe74643730be395018259ac1f4f5ee3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293709","number":293709,"commit":"82472eca44c309d1e263830e9914c16da0c40a8f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292975","number":292975,"commit":"93c362649e94387fa2a7513f1bec203ce6956ea1"}],"number":294108}
buildkite-pr-comment-->

kapral18 committed:
chore: restrict usage on enzyme imports in explore folders (attempt 3)

kapral18 commented on src/platform/packages/shared/kbn-cell-actions/.eslintrc.js:
this was the first draft and I completely missed the fact that we have root .eslintrc.js overrides 
with various file patterns that already match some of these patterns and by introducing 
overrides of the no-restricted-imports rule we silently override all those rules.

And also the fact that we introduce a new file pattern set here is not helping. It means even more fragmentation
of eslint rule overrides.

So this option was eventually ditched in favor of the next commit that introduces an attempt at hooking into already existing root .eslintrc.js overrides and redefine them with extended enzyme ban rule

kapral18 commented on src/platform/packages/shared/kbn-cell-actions/.eslintrc.js:
```diff
@@ -0,0 +1,60 @@
+      },
+    },
+  ],
+};
```
this was the first draft and I completely missed the fact that we have root .eslintrc.js overrides
with various file patterns that already match some of these patterns and by introducing
overrides of the no-restricted-imports rule we silently override all those rules.

And also the fact that we introduce a new file pattern set here is not helping. It means even more fragmentation
of eslint rule overrides.

So this option was eventually ditched in favor of the next commit that introduces an attempt at hooking into already existing root .eslintrc.js overrides and redefine them with extended enzyme ban rule

kapral18 commented on src/platform/packages/shared/kbn-cell-actions/.eslintrc.js:
```diff
@@ -7,54 +7,127 @@
+});
+
+module.exports = {
+  overrides: overridesWithNoRestrictedImportRule,
```
This is the second attempt. It almost worked. The problem with this was
very subtle but still pretty impactful.

Since I am redeclaring matching override blocks from root .eslintrc.js into nested .eslintrc.js I also move in the overrides[].files property

But the problem is those paths are manifesting the path from root config and they don't work with this nested config. Thus entire chunks of overrides are disabled because they never match.

Thus we come to attempt #3 with the next commit

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294355)
* Commit: 14d962e0af25bb4dc8b6a356b43ac857f25cecd2

### Failed CI Steps
* [Jest Tests #8](https://buildkite.com/elastic/kibana-pull-request/builds/294355#019645e8-23f3-4d03-8078-d0022893f8e2)
* [Jest Tests #8](https://buildkite.com/elastic/kibana-pull-request/builds/294355#0196460c-fcbe-4e7d-90c2-5e10f42053d6)
* [FTR Configs #28](https://buildkite.com/elastic/kibana-pull-request/builds/294355#019645e8-2421-4591-a0bb-ae3f50e32f6c)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294355#019645e8-23f3-4d03-8078-d0022893f8e2) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294355/jobs/019645e8-23f3-4d03-8078-d0022893f8e2/artifacts/0196460c-ccd4-4e3d-a143-45862928cc55) Jest Tests #<span></span>8 / HeaderSection it renders
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294355#0196460c-fcbe-4e7d-90c2-5e10f42053d6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294355/jobs/0196460c-fcbe-4e7d-90c2-5e10f42053d6/artifacts/0196461f-d445-4161-90b8-4419dddc1235) Jest Tests #<span></span>8 / HeaderSection it renders
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294355#019645e8-2421-4591-a0bb-ae3f50e32f6c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294355/jobs/019645e8-2421-4591-a0bb-ae3f50e32f6c/artifacts/019645f7-4f38-4934-bb6f-ff24386e4a6d) FTR Configs #<span></span>28 / management creating and deleting default data view validation can resolve errors and submit
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294355#019645e8-23f3-4d03-8078-d0022893f8e2) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294355/jobs/019645e8-23f3-4d03-8078-d0022893f8e2/artifacts/0196460c-ccd4-4212-929a-03e3e9110665) Jest Tests #<span></span>8 / Subtitle it renders
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/294355#0196460c-fcbe-4e7d-90c2-5e10f42053d6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/294355/jobs/0196460c-fcbe-4e7d-90c2-5e10f42053d6/artifacts/0196461f-d446-4a30-b098-9ee625593451) Jest Tests #<span></span>8 / Subtitle it renders

### History
* :broken_heart: [Build #294108](https://buildkite.com/elastic/kibana-pull-request/builds/294108) failed 290ca495abe74643730be395018259ac1f4f5ee3
* :broken_heart: [Build #293709](https://buildkite.com/elastic/kibana-pull-request/builds/293709) failed 82472eca44c309d1e263830e9914c16da0c40a8f
* :broken_heart: [Build #293499](https://buildkite.com/elastic/kibana-pull-request/builds/293499) failed 751a14756e426f39e1cc301279aba4c4ea2696e5
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294355","number":294355,"commit":"14d962e0af25bb4dc8b6a356b43ac857f25cecd2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294108","number":294108,"commit":"290ca495abe74643730be395018259ac1f4f5ee3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293709","number":293709,"commit":"82472eca44c309d1e263830e9914c16da0c40a8f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293147","number":293147,"commit":"e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a"}],"number":294355}
buildkite-pr-comment-->

kapral18 committed:
chore: fix ci regressions part 4

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294410)
* Commit: e8c7dc8b2be12181e02a555012d4f100c8e70ea4

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/5667c6cc435d8043dd088212e0430b833ce1a930) | [after](https://github.com/elastic/kibana/commit/e8c7dc8b2be12181e02a555012d4f100c8e70ea4) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 9.0MB | 9.0MB | +164.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/5667c6cc435d8043dd088212e0430b833ce1a930) | [after](https://github.com/elastic/kibana/commit/e8c7dc8b2be12181e02a555012d4f100c8e70ea4) | diff |
| --- | --- | --- | --- |
| `@kbn/cell-actions` | 1 | 3 | +2 |
| `@kbn/ecs-data-quality-dashboard` | 8 | 10 | +2 |
| `@kbn/security-solution-connectors` | 0 | 2 | +2 |
| `@kbn/security-solution-features` | 0 | 2 | +2 |
| `@kbn/security-solution-navigation` | 7 | 9 | +2 |
| `@kbn/security-solution-side-nav` | 2 | 4 | +2 |
| `@kbn/security-solution-upselling` | 1 | 3 | +2 |
| `@kbn/securitysolution-ecs` | 1 | 3 | +2 |
| `ecsDataQualityDashboard` | 5 | 7 | +2 |
| `securitySolution` | 574 | 583 | +9 |
| total |  |  | +27 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/5667c6cc435d8043dd088212e0430b833ce1a930) | [after](https://github.com/elastic/kibana/commit/e8c7dc8b2be12181e02a555012d4f100c8e70ea4) | diff |
| --- | --- | --- | --- |
| `@kbn/cell-actions` | 1 | 3 | +2 |
| `@kbn/ecs-data-quality-dashboard` | 8 | 10 | +2 |
| `@kbn/security-solution-connectors` | 0 | 2 | +2 |
| `@kbn/security-solution-features` | 0 | 2 | +2 |
| `@kbn/security-solution-navigation` | 7 | 9 | +2 |
| `@kbn/security-solution-side-nav` | 2 | 4 | +2 |
| `@kbn/security-solution-upselling` | 1 | 3 | +2 |
| `@kbn/securitysolution-ecs` | 1 | 3 | +2 |
| `ecsDataQualityDashboard` | 5 | 7 | +2 |
| `securitySolution` | 659 | 668 | +9 |
| total |  |  | +27 |


</details>


### History
* :broken_heart: [Build #294355](https://buildkite.com/elastic/kibana-pull-request/builds/294355) failed 14d962e0af25bb4dc8b6a356b43ac857f25cecd2
* :broken_heart: [Build #294108](https://buildkite.com/elastic/kibana-pull-request/builds/294108) failed 290ca495abe74643730be395018259ac1f4f5ee3
* :broken_heart: [Build #293709](https://buildkite.com/elastic/kibana-pull-request/builds/293709) failed 82472eca44c309d1e263830e9914c16da0c40a8f
* :broken_heart: [Build #293499](https://buildkite.com/elastic/kibana-pull-request/builds/293499) failed 751a14756e426f39e1cc301279aba4c4ea2696e5
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35
* :broken_heart: [Build #293147](https://buildkite.com/elastic/kibana-pull-request/builds/293147) failed e0d5e44e3c30ed1b20a71d96ccc8a7f2cdbd4a7a

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294410","number":294410,"commit":"e8c7dc8b2be12181e02a555012d4f100c8e70ea4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294355","number":294355,"commit":"14d962e0af25bb4dc8b6a356b43ac857f25cecd2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294108","number":294108,"commit":"290ca495abe74643730be395018259ac1f4f5ee3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293709","number":293709,"commit":"82472eca44c309d1e263830e9914c16da0c40a8f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293177","number":293177,"commit":"b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35"}],"number":294410}
buildkite-pr-comment-->

kapral18 committed:
Merge branch 'main' into chore/test/migrate-enzyme-to-rtl

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294561)
* Commit: 79a6e7988e9580ccde22ff5dee099f38284ca643

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/33993b7123bc0d6c85d9c42b15610cc0d5092281) | [after](https://github.com/elastic/kibana/commit/79a6e7988e9580ccde22ff5dee099f38284ca643) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 9.0MB | 9.0MB | +164.0B |

<details><summary>Unknown metric groups</summary>

#### ESLint disabled line counts
| id | [before](https://github.com/elastic/kibana/commit/33993b7123bc0d6c85d9c42b15610cc0d5092281) | [after](https://github.com/elastic/kibana/commit/79a6e7988e9580ccde22ff5dee099f38284ca643) | diff |
| --- | --- | --- | --- |
| `@kbn/cell-actions` | 1 | 3 | +2 |
| `@kbn/ecs-data-quality-dashboard` | 8 | 10 | +2 |
| `@kbn/security-solution-connectors` | 0 | 2 | +2 |
| `@kbn/security-solution-features` | 0 | 2 | +2 |
| `@kbn/security-solution-navigation` | 7 | 9 | +2 |
| `@kbn/security-solution-side-nav` | 2 | 4 | +2 |
| `@kbn/security-solution-upselling` | 1 | 3 | +2 |
| `@kbn/securitysolution-ecs` | 1 | 3 | +2 |
| `ecsDataQualityDashboard` | 5 | 7 | +2 |
| `securitySolution` | 574 | 583 | +9 |
| total |  |  | +27 |

#### Total ESLint disabled count
| id | [before](https://github.com/elastic/kibana/commit/33993b7123bc0d6c85d9c42b15610cc0d5092281) | [after](https://github.com/elastic/kibana/commit/79a6e7988e9580ccde22ff5dee099f38284ca643) | diff |
| --- | --- | --- | --- |
| `@kbn/cell-actions` | 1 | 3 | +2 |
| `@kbn/ecs-data-quality-dashboard` | 8 | 10 | +2 |
| `@kbn/security-solution-connectors` | 0 | 2 | +2 |
| `@kbn/security-solution-features` | 0 | 2 | +2 |
| `@kbn/security-solution-navigation` | 7 | 9 | +2 |
| `@kbn/security-solution-side-nav` | 2 | 4 | +2 |
| `@kbn/security-solution-upselling` | 1 | 3 | +2 |
| `@kbn/securitysolution-ecs` | 1 | 3 | +2 |
| `ecsDataQualityDashboard` | 5 | 7 | +2 |
| `securitySolution` | 659 | 668 | +9 |
| total |  |  | +27 |


</details>


### History
* :green_heart: [Build #294410](https://buildkite.com/elastic/kibana-pull-request/builds/294410) succeeded e8c7dc8b2be12181e02a555012d4f100c8e70ea4
* :broken_heart: [Build #294355](https://buildkite.com/elastic/kibana-pull-request/builds/294355) failed 14d962e0af25bb4dc8b6a356b43ac857f25cecd2
* :broken_heart: [Build #294108](https://buildkite.com/elastic/kibana-pull-request/builds/294108) failed 290ca495abe74643730be395018259ac1f4f5ee3
* :broken_heart: [Build #293709](https://buildkite.com/elastic/kibana-pull-request/builds/293709) failed 82472eca44c309d1e263830e9914c16da0c40a8f
* :broken_heart: [Build #293499](https://buildkite.com/elastic/kibana-pull-request/builds/293499) failed 751a14756e426f39e1cc301279aba4c4ea2696e5
* :broken_heart: [Build #293177](https://buildkite.com/elastic/kibana-pull-request/builds/293177) failed b83ccb7b0cefe7a0a0e4d5c37cdc158798a23f35

cc @kapral18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294561","number":294561,"commit":"79a6e7988e9580ccde22ff5dee099f38284ca643"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294410","number":294410,"commit":"e8c7dc8b2be12181e02a555012d4f100c8e70ea4"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294355","number":294355,"commit":"14d962e0af25bb4dc8b6a356b43ac857f25cecd2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294108","number":294108,"commit":"290ca495abe74643730be395018259ac1f4f5ee3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293709","number":293709,"commit":"82472eca44c309d1e263830e9914c16da0c40a8f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293499","number":293499,"commit":"751a14756e426f39e1cc301279aba4c4ea2696e5"}],"number":294561}
buildkite-pr-comment-->

kibanamachine commented:
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.19|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 218134
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kapral18 commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.19|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/218727">](https://github.com/elastic/kibana/pull/218727)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

kapral18 commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.18|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/219686">](https://github.com/elastic/kibana/pull/219686)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->

mistic commented:
This PR didn't make it into the latest 8.18.1 BC. Updating the labels.
